__tests__/app.test.js

```diff
@@ -165,10 +164,48 @@ describe("GET reviews ", () => {
             owner: expect.any(String),
             review_body: expect.any(String),
             review_img_url: expect.any(String),
-            created_at: expect.any(Number),
+            created_at: expect.any(String),
             votes: expect.any(Number),
+            comment_count: expect.any(Number),
           });
         });
       });
   });
+  it("200: should respond with an array sorted by date,descending", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.review).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  it("200: should be sorted by the correct key when passed a sort query", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .expect(200)
+      .then(({ body }) => {
+        body.review.forEach(() => {
+          expect.objectContaining({
+            title: expect.any(String),
+            designer: expect.any(String),
+            owner: expect.any(String),
+            review_img_url: expect.any(String),
+            review_body: expect.any(String),
+            category: "dexterity",
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
+  });
+
+  it("400: should return correct error when requested non existing sort_by", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=flyingspaghettimonster")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request!");
+      });
+  });
```

Comment:
Have a think about some extra tests:
* What happens if we look for a category that does not exist, what error code is this? what would we expect to see?
* What happens if we look for a category that DOES exist, but has no articles associated with it?